cmd/crank/xpkg/build.go

```diff
@@ -93,48 +93,53 @@ func (c *buildCmd) AfterApply() error {
+	ExamplesRoot             string   `short:"e" type:"path" help:"A directory of example YAML files to include in the package." default:"./examples"`
+	Ignore                   []string `placeholder:"PATH" help:"Comma-separated paths, specified relative to --package-root, to exclude from the package."`
+	Output                   string   `short:"o" type:"path" placeholder:"PATH" help:"The path to write the built xpkg file to."`
+	PackageRoot              string   `short:"f" type:"existingdir" help:"The directory that contains the package's crossplane.yaml file." default:"."`
+
+	// Internal state. These aren't part of the user-exposed CLI structure.
 	fs      afero.Fs
 	builder *xpkg.Builder
 	root    string
-
-	Output       string   `optional:"" short:"o" help:"Path for package output."`
-	PackageRoot  string   `short:"f" help:"Path to package directory." default:"."`
-	ExamplesRoot string   `short:"e" help:"Path to package examples directory." default:"./examples"`
-	Ignore       []string `help:"Paths, specified relative to --package-root, to exclude from the package."`
-
-	EmbedRuntimeImage   string `help:"An OCI image reference to the package's runtime. The package will embed this image." placeholder:"\"example/runtime-image:latest\"" xor:"runtime-image"`
-	EmbedRuntimeTarball string `help:"An OCI image tarball of the package's runtime. The package will embed this image." placeholder:"\"example-runtime-image.tar\"" type:"existingfile" xor:"runtime-image"`
 }
 
 func (c *buildCmd) Help() string {
 	return `
-The build command creates a Crossplane package from the local filesystem. A
-package is an OCI image containing metadata and configuration manifests that can
-be used to extend Crossplane with new functionality.
+Crossplane can be extended using packages. A Crossplane package is sometimes
+called an xpkg. Crossplane supports configuration, provider and function
+packages. 
+
+A package is an opinionated OCI image that contains everything needed to extend
+Crossplane with new functionality. For example installing a provider package
+extends Crossplane with support for new kinds of managed resource (MR).
 
-Crossplane supports configuration, provider and function packages. 
+This command builds a package from a local directory of files.
 
-Packages can embed a runtime image. When a package embeds a runtime image
-Crossplane can use the same OCI image to install and run the package. For
-example a provider package can embed the provider's controller image (its
-"runtime"). Crossplane will then use the package as the provider pod's image.
+Provider and function packages can embed a runtime image. When a package embeds
+a runtime image Crossplane can use the same OCI image to install and run the
+package. For example a provider package can embed the provider's controller
+image (its "runtime"). Crossplane will then use the package as the provider
+pod's image.
```

Comment:
should we reject trying to build images with and embed image if the meta spec defines another image at `spec.image`?

Reply:
Probably a good idea, but let's tackle that separately.